checkov/common/output/record.py

```diff
@@ -15,6 +15,7 @@
 
 DEFAULT_SEVERITY = "none"  # equivalent to a score of 0.0 in the CVSS v3.0 Ratings
 
+output_violation_len = os.environ.get('CHECKOV_OUTPUT_VIOLATION_LENGTH', 50)
```

Comment:
```suggestion
OUTPUT_VIOLATION_LEN = force_int(os.getenv('CHECKOV_OUTPUT_VIOLATION_LENGTH') or 50
```
since it is static, you usually use upper case for the name.
if an env var is used, then it is always a string, we have a function which forces an int cast, if it fails then `None` is returned. In that case the `or` part will change it to 50 🙂 and don't forget to import the function 😄